main/integration-tests/__test__/api-tests/common/migration/my-study-migration.test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../../support/setup');
+const errorCode = require('../../../../support/utils/error-code');
+
+describe('Migration for My Studies scenarios', () => {
+  let setup;
+  let adminSession;
+  let defaultUser;
+  let defaultUserSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+    // Create cognito user to migrate ownership to
+    // Proper way that currently doesn't work:
+    // const username = setup.gen.username();
+    // defaultUser = adminSession.resources.users.defaults({ username });
+    // defaultUser = await await adminSession.resources.users.create(defaultUser);
+    // await adminSession.resources.users.user(defaultUser.uid);
+    // Workaround:
+    defaultUserSession = await setup.createResearcherSession();
+    defaultUser = defaultUserSession.user;
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Getting My Studies', () => {
+    it('should fail if request not by admin', async () => {
+      const researcherSession = await setup.createResearcherSession();
+      await expect(researcherSession.resources.migration.listInternaAuthUserMyStudies()).rejects.toMatchObject({
+        code: errorCode.http.code.forbidden,
+      });
+    });
+
+    it('should return message that there is nothing to migrate', async () => {
+      await expect(adminSession.resources.migration.listInternaAuthUserMyStudies()).resolves.toStrictEqual(
+        'No My Studies are owned my internal users. No migration needed.',
+      );
+    });
+  });
+
+  describe('Migrating My Studies', () => {
+    let studyId;
+
+    it('should fail if request not by admin', async () => {
+      const researcherSession = await setup.createResearcherSession();
+      // Create My Study
+      studyId = setup.gen.string({ prefix: `create-my-study-test` });
+      await researcherSession.resources.studies.create({ id: studyId, name: studyId, category: 'My Studies' });
+      await researcherSession.resources.studies.study(studyId);
```

Comment:
what does this line do? It gets the study, but doesn't store the value of the study?